CMakeLists.txt

```diff
@@ -77,6 +77,8 @@ if (WIN32)
     set(PLATFORM_LIBS Secur32 Crypt32)
     set(TLS_STACK_DETERMINED ON)
 
+    set(EVENT_LOOP_DEFINE "IO_COMPLETION_PORTS")
```

Comment:
You're always setting this whether or not the USE_IO_COMPLETION_PORT options is ON